sklearn/ensemble/_bagging.py

```diff
@@ -242,7 +242,8 @@ def fit(self, X, y, sample_weight=None):
         """
         return self._fit(X, y, self.max_samples, sample_weight=sample_weight)
 
-    def _parallel_args(self):
+    @staticmethod
```

Comment:
Functionally this is the same, but I prefer the instance method just in case we have `parallel_args` that depend on `self`. May we revert this change?